(Add) Simple Example for String Concatenation (#6909)
harness/developer-hub: docs/platform/variables-and-expressions/harness-variables.md

<+<+stage.variables.v1> != "dev">
```

> There may be situations when a string needs to be concatenated into a variable, to allow it to be utilized utilized as an expression.  In these situations, look to add double quotes around the string.

Reviewer: ```suggestion
There might be situations when a string needs to be concatenated into a variable to allow it to be utilized as an expression. In these situations, you must add double quotes around the string.
```
